inst/shiny-server/server/upload.R

```diff
@@ -6,65 +6,96 @@ DataList <- reactiveValues(data = DataSetList())
 # set up list of datasets (scan the repository, looking for .rds files)
 observe({
   repo_dir <<- get_repo_location()
-  rds_files <- list.files(repo_dir, pattern = '.rds') %>% sub('\\.rds$', '', .)
-
+  if (input$respository.type == 'PBO') {
+    rds_files <- list.files(repo_dir, pattern = '.rds') %>% sub('\\.rds$', '', .)
+    rds_files <- c(rds_files, "Example_small", "Example_large")
+  }
+  else if (input$respository.type == 'NEVERGRAD'){
+    rds_files <- list.files(paste0(repo_dir, "/nevergrad"), pattern = '.rds') %>% sub('\\.rds$', '', .)
+  }
+  
   if (length(rds_files) != 0) {
-    updateSelectInput(session, 'repository.dataset', choices = c(rds_files, "Example_small", "Example_large"), selected = NULL)
+    updateSelectInput(session, 'repository.dataset', choices = rds_files, selected = NULL)
   } else { # TODO: the alert msg should be updated
     shinyjs::alert("No repository file found. To make use of the IOHProfiler-repository,
                    please create a folder called 'repository' in your home directory
                    and make sure it contains at least one '.rds'-file, such as the ones
                    provided on the IOHProfiler github-page.")
-    updateSelectInput(session, 'repository.dataset', choices = c("Example_small", "Example_large"), selected = NULL)
+    updateSelectInput(session, 'repository.dataset', choices = NULL, selected = NULL)
   }
-  })
+})
+
+
+observeEvent(input$repository.type, {
+  req(input$repository.type)
+  if (input$respository.type == 'PBO') {
+    names <- list.files(repo_dir, pattern = '.rds') %>% sub('\\.rds$', '', .)
```

Comment:
Do we want to create a folder called `PBO` and dump all relevant rds file there? :)
The data management is another point we need to discuss.

Reply:
Yes, lets discuss this when you are back in Leiden